nextmv/cloud/client.py

```diff
@@ -180,6 +180,39 @@ def request(
 
         return response
 
+    def upload_to_presigned_url(
```

Comment:
Why do we need this new method? Can't we use the one we already have? What is the main issue tht would prevent us from using it?

On another note, this method doesn't use "self", should it really be a class method?

One drawback of not using the other method is that we have no timeouts, retries, backoff, or anything.

Reply:
We talked about this offline and will aim to keep it with the existing logic as much as possible.